runhttpd.sh

```diff
@@ -13,11 +13,25 @@ mkdir -p /shared/html
 chmod 0777 /shared/html
 
 if [[ $IRONIC_FAST_TRACK == true ]]; then
-    INSPECTOR_EXTRA_ARGS="ipa-api-url=http://${IRONIC_URL_HOST}:6385"
+    if [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
+        INSPECTOR_EXTRA_ARGS=" ipa-api-url=https://${IRONIC_URL_HOST}:6385"
+    else
+        INSPECTOR_EXTRA_ARGS=" ipa-api-url=http://${IRONIC_URL_HOST}:6385"
+    fi
 else
     INSPECTOR_EXTRA_ARGS=""
 fi
 
+if [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
+    INSPECTOR_EXTRA_ARGS="${INSPECTOR_EXTRA_ARGS} ipa-inspection-callback-url=https://${IRONIC_URL_HOST}:5050/v1/continue"
+else
+    INSPECTOR_EXTRA_ARGS="${INSPECTOR_EXTRA_ARGS} ipa-inspection-callback-url=http://${IRONIC_URL_HOST}:5050/v1/continue"
+fi
+
+if [ ! -z "$IPA_INSECURE" ]; then
```

Comment:
This should be unconditional until we have at least one way to pass certificates to the ramdisk (tracked in https://storyboard.openstack.org/#!/story/2007214)

Reply:
ok